pkg/types/cloud_providers.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright © 2021 Rasa Technologies GmbH
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package types
+
+type CloudProvider string
+
+const (
+	CloudProviderUnknown      CloudProvider = "Unknown"
+	CloudProviderGoogle       CloudProvider = "GCP"
+	CloudProviderAmazon       CloudProvider = "AWS"
+	CloudProviderAlibaba      CloudProvider = "Alibaba Cloud"
+	CloudProviderAzure        CloudProvider = "Azure"
+	CloudProviderDigitalOcean CloudProvider = "Digital Ocean"
+	CloudProviderOracle       CloudProvider = "Oracle Cloud"
```

Comment:
I believe you can remove the `CloudProvider` from the second entry to the last one, and it should infer they are all type `CloudProvider`.